packages/compass-connections/src/components/connections.tsx

```diff
@@ -99,12 +99,13 @@ function Connections({
     dataService: DataService
   ) => void;
   isConnected: boolean;
-  connectionStorage?: typeof ConnectionStorage;
   appName: string;
   getAutoConnectInfo?: () => Promise<ConnectionInfo | undefined>;
   connectFn?: ConnectFn;
 }): React.ReactElement {
   const { log, mongoLogId } = useLoggerAndTelemetry('COMPASS-CONNECTIONS');
+  const connectionStorage = connectionStorageLocator();
```

Comment:
Locators are a pattern to be used with the plugin service injection mechanism, not directly inside components code. If this is temporary before we refactor this to a proper plugin, let's add a TODO with a relevant ticket number here: https://jira.mongodb.org/browse/COMPASS-7397

Reply:
Done.